### DISCUSSION TITLE
Blogpost 1

### DISCUSSION THREAD

**Discussion by ChristophNie** [LeonRother/SE_Platzverwaltung#2] [General]
# Teamvorstellung

- Jannik 
- Moritz
- Leon 
- Hagen 
- Christoph

# Produktvision
Unser Platzverwaltungssystem soll es ermöglichen einfach und schnell Sitzplätze zu reservieren. Nutzer sollen die Verfügbarkeit in Echtzeit sehen und können ihre Buchungen mühelos verwalten. Der Nutzer bekommt automatisch eine Bestätigung.

# Techniken/Plattform

Für das Backend wird auf **c# asp.net** gesetzt. Für das Frontend wird auf **Vue.js Javascript/Typescript** gesetzt.
Datenbank **tbd**

# Scrum Setup

Zum planen des Projekts wird YouTrack verwendet. Am Ende eines Sprints wird ein Review sowie ein Retro abgehalten. Die Sprintdauer wird noch festgelegt.

# Scrum Board
![image](https://github.com/user-attachments/assets/9719ae03-c4e5-4062-b07a-c5b100a10ccc)

**Comment by SchloesserJonas** [comment]
Hi,

Das klingt doch schon mal nach einer guten Idee!

Wie plant ihr den Faktor "Echtzeit" umzusetzen?
Das Senden von Daten von Server zu Client ist ja ohne vorherige Clientanfrage nicht möglich...
Überlegt vielleicht, ob ihr die Funktionalität streicht und nach z.B. 30 Sekunden Inaktivität ein Popup anzeigt und den Nutzer auffordert einen page-reload zu machen.
Alternativ könnt ihr die Verwendung von Websockets in Erwägung ziehen um diese Funktion zu realisieren.

Wo liegt der genaue Anwendungsfall für eure Software?
Restaurants, Veranstaltungen, etc...
Wie plant ihr die Sitzplätze entsprechend der Umgebung zu visualisieren?

Gefällt uns gut!
Wie sind eure Aufgaben auf die Teammitglieder verteilt? 

GaLiGrü 😊
Paula, Safae, Samuel, Valentin, Jonas
> Green Sprout 🌱

**Comment by alex-3709537** [comment]
Hallo,

die Vision, ein einfach zu bedienendes Platzverwaltungssystem zu schaffen, das Sitzplatzreservierungen in Echtzeit ermöglicht, ist klar und kundenorientiert.
Die Nutzerfreundlichkeit steht im Fokus, was ein wichtiger Erfolgsfaktor ist. Besonders die automatische Bestätigung der Buchung klingt nach einer komfortablen und zeitgemäßen Lösung. Ihr habt hier eine klare und überzeugende Vision formuliert. Sehr gut gemacht!

Gruß
>Restaurant Reservierung 🗓️

**Comment by LstPokey** [comment]
Hallöchen, 
Das Projekt hat eine klare Produktvision für ein Platzverwaltungssystem. Die Wahl von C# ASP:NET für das Backend und vue.js mit JavaScript/Typescript für das Frontend ist zeitgemäß und robust. Das Scrum-Setup zeigt ein gut strukturiertes agiles Vorgehen. Insgesamt zeigt sich das Projekt gut struckturiert und geplant. 
LG Oskar
